src/EditorFeatures/Core/Implementation/Classification/SyntacticClassificationTaggerProvider.cs

```diff
@@ -32,10 +39,18 @@ internal partial class SyntacticClassificationTaggerProvider : ITaggerProvider
         public SyntacticClassificationTaggerProvider(
             IForegroundNotificationService notificationService,
             ClassificationTypeMap typeMap,
+            [Import(AllowDefault = true)] IViewSupportsClassificationService viewSupportsClassificationService,
```

Comment:
Why? Can we flow "Opt" through all of the users here to make it clear that the service can be null?